src/DBFTables.jl

```diff
@@ -133,7 +135,7 @@ end
 
 function read_dbf(io::IO; deleted=false)
     header = read_dbf_header(io)
-	df = DataFrame(map(f->f.typ, header.fields), map(f->Symbol(f.nam), header.fields), 0)
+	df = DataFrame(map(f->Union{f.typ,Missing}, header.fields), map(f->Symbol(f.nam), header.fields), 0)
```

Comment:
Why does this have to change?

Reply:
For columns with logical values, it might be `missing` when the values is not `in ['Y', 'y', 'T', 't']` nor 'in ['N', 'n', 'F', 'f']'.
As for numeric columns, I have encountered file with numeric field that is not parse-able such as `****`.
So I thought just make every columns to anticipate field that could be missing.